app/services/process_submission_service.rb

```diff
@@ -1,16 +1,14 @@
-class ProcessSubmissionService # rubocop:disable  Metrics/ClassLength
+class ProcessSubmissionService
   attr_reader :submission_id
 
   def initialize(submission_id:)
     @submission_id = submission_id
+    @submission = Submission.find(submission_id)
+    @payload_service = SubmissionPayloadService.new(submission.payload)
   end
 
   # rubocop:disable Metrics/MethodLength
   def perform # rubocop:disable Metrics/AbcSize
-    submission.update_status(:processing)
```

Comment:
not for this PR but i'm hoping this column can be killed soon

Reply:
Noted